tests/partitions_test.py

```diff
@@ -87,8 +87,10 @@ def test_partition_majority(network, args):
 
 
 @reqs.description("Isolate primary from one backup")
+@reqs.exactly_n_nodes(3)
 def test_isolate_primary_from_one_backup(network, args):
-    primary, backups = network.find_nodes()
+    p, backups = network.find_nodes()
+    b_0, b_1 = backups
```

Comment:
Could we call these `p_0`, `b_1` and `b_2` so that it matches the `local_node_id`s in the infra?

Reply:
It probably doesn't, since previous tests have caused multiple partitions and elections. These names are completely local to the test - happy to change them to `a, b, c` if that's easier, just want to know precisely which node we're talking about.

Reply:
Ah yes, these tests are chained, even on the non-suite runners.